testUsers.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+ROLES=(ADMIN PM TRANSLATOR USER)
```

Comment:
Can this be moved into a Java integration test. If not, this probably should be still moved to the "test" directory

Reply:
I have removed the script.